model/otel/metrics.yaml

```diff
@@ -110,3 +110,30 @@ groups:
           recommended: when applicable
       - ref: error.type
         examples: ["rejected", "timeout", "500", "java.net.UnknownHostException"]
+
+  - id: metric.otel.sdk.exporter.http.request.duration
```

Comment:
Any reason for limiting this metric to `http` instead of defining it for all network-request based exporters?
E.g. `otel.sdk.exporter.request.duration` instead of `otel.sdk.exporter.http.request.duration`? I think we could still keep the `http.response.status_code` around with `recommended: when applicable`.

Reply:
No strong reason in particular. I just wanted the status code to be represented as an attribute.